### DISCUSSION TITLE
It is possible to custom the labels of Feature Overview, Features and Scenarios?

### DISCUSSION THREAD

**Discussion by rushian** [WasiqB/multiple-cucumber-html-reporter#334] [Ideas]
I'm trying to translate the report into Portuguese, but I don't know how to manipulate this data...

**Comment by WasiqB** [comment]
Hi @rushian ,

Thanks for sharing your idea about supporting different languages.

I will prioritise this feature soon.

In the meantime, try out the new version [4.0.0-beta.0](https://github.com/WasiqB/multiple-cucumber-html-reporter/releases/tag/v4.0.0-beta.0) which was released yesterday.
